modules/imgur.py

```diff
@@ -150,3 +150,33 @@ def imgur(self, event):
             event["stdout"].write(result)
         else:
             raise utils.EventResultsError()
+            
+    @utils.hook("received.command.rehost", channel_only=True)
+    @utils.kwarg("permission", "rehost")
+    def rehost(self, event):
+        channel = event["target"]
+        args = event["args_split"]
+
+        if len(args) is not 1:
+            event["stderr"].write("You must provide exactly one link to rehost.")
+            return
+
+        url = args[0]
+
+        page = utils.http.request(
+            URL_REHOST,
+            method="POST",
+            headers={"Authorization": "Basic %s" % self.bot.config["imgur-api-key"]},
+            post_data={"image": url, "type": "url"},
+        ).json()
+
+        if page["success"] is not True:
+            event["stderr"].write("Image upload successful")
+            return
```

Comment:
If `success` is not true, then why are we saying that the upload was successful?